tests/contracts/config.py

```diff
@@ -0,0 +1,16 @@
+from abc import ABC, abstractmethod
+
+
+class Contract(ABC):
+
+    @abstractmethod
+    def test_default(self, mocked_config):
+        pass
+
+    @abstractmethod
+    def test_repo(self, mocked_config):
+        pass
+
+    @abstractmethod
+    def test_user(self, mocked_config):
+        pass
```

Comment:
These properties have become private and should not be tested.
The other public props should be tested eg. force_checklist.

Reply:
removed these tests, updated contract, updated any tests that subsequently failed the contract